lib/SampleService/core/api_translation.py

```diff
@@ -612,3 +612,24 @@ def links_to_dicts(links: List[DataLink]) -> List[Dict[str, Any]]:
             'expired': ex
         })
     return ret
+
+def update_samples_acls(
+    params, samples_client, user_lookup, user, token, perms, log_info
+):
+    acldelta = acl_delta_from_dict(params)
+    admin = check_admin(
+        user_lookup,
+        token,
+        perms,
+        "update_sample_acls",
+        log_info,
+        skip_check=not params.get('as_admin')
+    )
+    ids = params.get("ids")
```

Comment:
Should check ids in a similar way to https://github.com/kbase/sample_service/blob/master/lib/SampleService/core/api_translation.py#L83
... and fail early if an ID is bad or missing

Reply:
Now [validates ids](https://github.com/kbase/sample_service/pull/421/files#diff-bcd1950c61ccf664320d98b81c759b2fae5eeccfe183ed541236b6f4defde6faR52) using the new [`validate_sample_id`](https://github.com/kbase/sample_service/pull/421/files#diff-69b73db1b851f478d30aa6c4fc55c504d1c6c0074b5487a50dcbb047818a6ed0R610) function.